sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusScope.cs

```diff
@@ -28,29 +16,15 @@ namespace Azure.Messaging.ServiceBus.Tests
     ///
     public static class ServiceBusScope
     {
-        /// <summary>The maximum number of attempts to retry a management operation.</summary>
-        private const int RetryMaximumAttempts = 20;
-
-        /// <summary>The number of seconds to use as the basis for backing off on retry attempts.</summary>
-        private const double RetryExponentialBackoffSeconds = 3.0;
-
-        /// <summary>The number of seconds to use as the basis for applying jitter to retry backoff calculations.</summary>
-        private const double RetryBaseJitterSeconds = 60.0;
-
-        /// <summary>The buffer to apply when considering refreshing; credentials that expire less than this duration will be refreshed.</summary>
-        private static readonly TimeSpan CredentialRefreshBuffer = TimeSpan.FromMinutes(5);
-
-        /// <summary>The random number generator to use for each requesting thread.</summary>
-        private static readonly ThreadLocal<Random> RandomNumberGenerator = new ThreadLocal<Random>(() => new Random(Interlocked.Increment(ref s_randomSeed)), false);
-
-        ///<summary>The URI for the location of the resource manager for the active cloud environment.</summary>
-        private static readonly Uri ResourceManagerUri = new Uri(ServiceBusTestEnvironment.Instance.ResourceManagerUrl);
-
-        /// <summary>The seed to use for random number generation.</summary>
-        private static int s_randomSeed = Environment.TickCount;
-
-        /// <summary>The token credential to be used with the  Service Bus management client.</summary>
-        private static ManagementToken s_managementToken;
+        /// <summary> The client used to create and delete resources on the Service Bus namespace. </summary>
+        private static ServiceBusAdministrationClient CreateClient(string namespaceName)
```

Comment:
Should we make this a `Lazy<ServiceBusAdministrationClient` instance so that it we don't create it more than necessary, or is there benefit to having the independent instances?

Reply:
The reason for not caching it is that some tests are passing the namespace override parameter so we need multiple different clients. We could cache per namespace, but I didn't think it was worth the extra complexity.